### ISSUE TITLE
Install fails due to build error

### ISSUE THREAD

**Issue by maciej** [cloudflare/cloudflared#88]
``` 
⨯ go install github.com/cloudflare/cloudflared/cmd/cloudflared
go: finding github.com/cloudflare/cloudflared/cmd/cloudflared latest
[...]
# github.com/cloudflare/cloudflared/tunneldns
../../go/pkg/mod/github.com/cloudflare/cloudflared@v0.0.0-20190403193134-980bee22a50c/tunneldns/metrics.go:40:13: cannot use ctx (type context.Context) as type string in argument to vars.Report
```

```
⨯ go version
go version go1.12.4 darwin/amd64
```

**Comment by sssilver** [comment]
Thanks, this is due to  the CoreDNS dependency being pointed at `master`. We'll pin it.

**Comment by jbergstroem** [comment]
Fixed in bab7583a97e8f7776640889f8b823f63a77e709a?

**Comment by paulquerna-okta** [comment]
Fixed by https://github.com/cloudflare/cloudflared/pull/94
